geonode/ogpsearch/utils.py

```diff
@@ -49,10 +50,7 @@ def get_domain(url):
         hostname = urlParts.hostname
         if hostname == "localhost":
             return "Harvard" # assumption
-        domainParts = hostname.split(".")
-        if len(domainParts) == 1:
-            return domainParts[0]
-        return domainParts[-2].capitalize()
+        return hostname
```

Comment:
From the code base it was being triggered at https://github.com/cga-harvard/cga-worldmap/blob/ogpsearch2/geonode/ogpsearch/utils.py#L98, the same file.